packages/driver/cypress/integration/e2e/multi-domain/multi_domain_validation_specs.ts

```diff
@@ -42,7 +42,7 @@ describe('multi-domain', () => {
     })
 
     it('succeeds on a complete origin', () => {
-      cy.origin('http://foobar1.com:3500', () => {})
+      cy.origin('http://foobar1.com:3500', () => undefined)
```

Comment:
Just curious - why change these from `{}` to `undefined`?

Reply:
I was just updating to a single pattern in this file.